README.md

```diff
@@ -108,205 +110,49 @@ replicaset.apps/velero-76546b65c8          1         1         1         2m17s
 velero client config set namespace=oadp-operator
 ```
 
-### Configure Velero Plugins
-
-There are mainly two categories of velero plugins that can be specified while installing Velero:
-
-1. `default-velero-plugins`:<br>
-   4 types of default velero plugins can be installed - AWS, GCP, Azure and OpenShift. For installation, you need to specify them in the `konveyor.openshift.io_v1alpha1_velero_cr.yaml` file during deployment.
-   ```
-    apiVersion: konveyor.openshift.io/v1alpha1
-    kind: Velero
-    metadata:
-      name: example-velero
-    spec:
-      default_velero_plugins:
-      - azure
-      - gcp
-      - aws
-      - openshift    
-   ```
-   The above specification will install Velero with all the 4 default plugins.
-   
-2. `custom-velero-plugin`:<br>
-   For installation of custom velero plugins, you need to specify the plugin `image` and plugin `name` in the `konveyor.openshift.io_v1alpha1_velero_cr.yaml` file during deployment.
-
-   For instance, 
-   ```
-    apiVersion: konveyor.openshift.io/v1alpha1
-    kind: Velero
-    metadata:
-      name: example-velero
-    spec:
-      default_velero_plugins:
-      - azure
-      - gcp
-      custom_velero_plugins:
-      - name: custom-plugin-example
-        image: quay.io/example-repo/custom-velero-plugin   
-   ```
-   The above specification will install Velero with 3 plugins (azure, gcp and custom-plugin-example).
+***
+## Customize Installation
+***
 
-### Enable CSI plugin for Velero
+### Plugin customization
 
-By default the CSI plugin is not enabled, in order to enable the [CSI plugin](https://github.com/vmware-tanzu/velero-plugin-for-csi/) for velero, you need to specify a flag `enable_csi_plugin` and set it to `true` in the `konveyor.openshift.io_v1alpha1_velero_cr.yaml` file during the installation.
+During the Velero installation using OADP Operator we need at least one plugin. Please refer [Velero plugin customization](docs/plugins.md) for more details.
```

Comment:
Nit: The Velero installation requires at least one cloud provider plugin installed. Please refer ...

Reply:
Done, updated the readme